File: lib-content-scrapers/src/main/java/com/ustadmobile/lib/contentscrapers/voa/VoaResponse.java
package com.ustadmobile.lib.contentscrapers.voa;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class VoaResponse {

    @Expose
    @SerializedName("articleSection")
    public String articleSection;
    @Expose
    @SerializedName("description")
    public String description;

    @Expose
    @SerializedName("keywords")
    public String keywords;
    @Expose
    @SerializedName("dateModified")
    public String dateModified;
    @Expose
    @SerializedName("datePublished")
    public String datePublished;

    @Expose
    @SerializedName("name")
    public String name;
    @Expose
    @SerializedName("headline")
    public String headline;
    @Expose
    @SerializedName("url")
    public String url;
    @Expose
    @SerializedName("mainEntityOfPage")
    public String mainEntityOfPage;
    @Expose
    @SerializedName("inLanguage")
    public String inLanguage;

}

File: lib-content-scrapers/src/main/kotlin/com/ustadmobile/lib/contentscrapers/voa/VoaResponse.kt
package com.ustadmobile.lib.contentscrapers.voa

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class VoaResponse {

    @Expose
    @SerializedName("articleSection")
    var articleSection: String? = null
    @Expose
    @SerializedName("description")
    var description: String? = null

    @Expose
    @SerializedName("keywords")
    var keywords: String? = null
    @Expose
    @SerializedName("dateModified")
    var dateModified: String? = null
    @Expose
    @SerializedName("datePublished")
    var datePublished: String? = null

    @Expose
    @SerializedName("name")
    var name: String? = null
    @Expose
    @SerializedName("headline")
    var headline: String? = null
    @Expose
    @SerializedName("url")
    var url: String? = null
    @Expose
    @SerializedName("mainEntityOfPage")
    var mainEntityOfPage: String? = null
    @Expose
    @SerializedName("inLanguage")
    var inLanguage: String? = null

}
